examples/coco/.tensorleap/dataset.py

```diff
@@ -17,7 +17,7 @@
 PROJECT_ID = 'example-dev-project-nmrksf0o'
 
 image_size = 128
-categories = ['person', 'car']
+categories = ['person', 'car', 'background']
```

Comment:
This will not work IMO, since line 67 will try to get the images of a 'background'  class and will raise an error.
This should be ['person', 'car'] and labels at line 235 should be ['background', 'person', 'car'].